packages/api/controllers/nomination.js

```diff
@@ -14,10 +14,22 @@ const getNominationById = async (req, res) => {
     if (nomination) {
       return res.status(200).json({ nomination });
     }
-    return res.status(404).send('Nomination with the specified ID does not exist!');
+    return res
+      .status(404)
+      .send('Nomination with the specified ID does not exist!');
   } catch (error) {
     return res.status(500).send(error.message);
   }
 };
 
-module.exports = { getNominationById };
+const createNomination = async (req, res) => {
+  try {
+    const nomination = await db.Nomination.create(req.body);
+
+    return res.status(201).json({});
```

Comment:
I think we should return the nomination on the response instead of an empty object.